runtime/manta/src/weights/pallet_parachain_staking.rs

```diff
@@ -84,44 +84,44 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_parachain_staking::WeightInfo for SubstrateWeight<T> {
     // Storage: ParachainStaking InflationConfig (r:1 w:1)
     fn set_staking_expectations() -> Weight {
-        (16_255_000 as Weight)
+        (45_727_000 as Weight)
```

Comment:
this is a big increase

Reply:
artifact of the benchmarking run. Second run has
```
    fn set_staking_expectations() -> Weight {
        (16_517_000 as Weight)
        ```